system/jlib/jexcept.cpp

```diff
@@ -1607,13 +1613,42 @@ void jlib_decl *setSEHtoExceptionHandler(IExceptionHandler *handler)
     return ret;
 }
 
+#if defined(__linux__)
+  static timer_t killTimerId;
+  enum KillTC { UNINIT, CREATED };
+  static std::atomic<KillTC> killTimerCreated { UNINIT };
+  static CriticalSection killTimerCS;
+#endif
+
 void jlib_decl enableSEHtoExceptionMapping()
 {
 #ifdef NOSEH
     return;
 #endif
     if (SEHnested++)
         return; // already done
+
+#if defined(__linux__)
+    KillTC state = killTimerCreated.load();
```

Comment:
minor: Because this function is called very rarely I don't think it needs the complication of checking outside the critical section.

Reply:
ok, I agree